src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy-details/rgw-multisite-sync-policy-details.component.ts

```diff
@@ -136,22 +138,17 @@ export class RgwMultisiteSyncPolicyDetailsComponent implements OnChanges {
       click: () => this.openModal(FlowType.directional),
       canBePrimary: (selection: CdTableSelection) => !selection.hasSelection
     };
-    const dirEditAction: CdTableAction = {
-      permission: 'update',
-      icon: Icons.edit,
-      name: this.actionLabels.EDIT,
-      click: () => this.openModal(FlowType.directional, true),
-      disable: () => true // TODO: disabling 'edit' as we are not getting flow ID from backend which is needed for edit
-    };
     const dirDeleteAction: CdTableAction = {
       permission: 'delete',
       icon: Icons.destroy,
-      disable: () => true, // TODO: disabling 'delete' as we are not getting flow ID from backend which is needed for deletion
+      // TODO: disabling 'delete' as we are not getting flow_id from backend which is needed for deletion
+      disable: () =>
+        'Deleting the directional flow is disabled in the UI. Please use CLI to delete the directional flow',
```

Comment:
cc @nizamial09 & I discussed earlier about this , to have a tooltip stating the above or completely removing the delete action.

Reply:
yeah, usually if something action is disabled we just show them in the action itself rather than making a bigger noise and banner. So I guess we can follow the same thing here.. And more over I don't think its good to show the alert in a banner because this is an unfortunate limitation (a CLI issue) so probably good to dial down the noise.

Reply:
@nmunet @nizamial09 my suggestion was to use the text we had earlier. But looking again at this, I am fine with this one.

Reply:
Or shall we just remove the action itself? @afreen23 
users might wonder why we are showing disabled button with no tooltip!

Reply:
i am okay with either removing those actions or just keeping it with the tooltip. Otherwise it'd be confusing.

Reply:
resolving this one and keeping the button as it is with tooltip.